nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -168,6 +168,8 @@ Use `configure.packages` instead.
 
 - The `diamond` package has been update from 0.8.36 to 2.0.15. See the [upstream release notes](https://github.com/bbuchfink/diamond/releases) for more details.
 
+- `services.klipper` will now use `/var/lib/klipper` as its default configuration folder, and will populate `/var/lib/klipper/gcodes` for gcode storage.
+
```

Comment:
IMO, this needs to be rewritten to mention `stateVersion`, otherwise it's hard to grep for. Here's an example of a good one from 22.05's manual:
```
- For new installations `virtualisation.oci-containers.backend` is now set to `podman` by default.
  If you still want to use Docker on systems where `system.stateVersion` is set to to `"22.05"` set `virtualisation.oci-containers.backend = "docker";`.Old systems with older `stateVersion`s stay with "docker".
```

Reply:
Oh god, that logic is even more perverted than I expected. I guess I'll just drop the note, and leave the config as it is and where it is if `settings` are specified. 
![image](https://user-images.githubusercontent.com/6453661/187042724-9823474e-1fed-4637-9649-1c7fd28ebaf2.png)

Reply:
gooood, okay

that removes ability to specify initial config with nix options, which is not good
so I am adding `isConfigMutable`, and explicitly naming options — or it will become undecipherable ;D